### PR TITLE
dist: support nonroot and offline mode for scylla-housekeeping

### LINKED ISSUE #13084: Support scylla-housekeeping in nonroot mode
Currently we disabled scylla-housekeeping for nonroot mode since we don't have "repo file" for offline installation (relocatable pakcage), but it likely work with nonroot by running without --repo-files parameter, since looks like repo_id and repo_type are not mandatory parameters (these variables are extracted from repository url):
https://github.com/scylladb/scylladb/blob/master/dist/common/scripts/scylla-housekeeping#L134

looks like it just works:
```
$ sudo -u scylla /opt/scylladb/scripts/scylla-housekeeping --uuid-file /var/lib/scylla-housekeeping/housekeeping.uuid -q -c /etc/scylla.d/housekeeping.cfg version --mode r
/opt/scylladb/scripts/libexec/scylla-housekeeping:184: DeprecationWarning: The SafeConfigParser class has been renamed to ConfigParser in Python 3.2. This alias will be removed in future versions. Use ConfigParser directly instead.
  config = configparser.SafeConfigParser()
You current Scylla release is 2021.1.17 the latest minor release is 2022.2.0 go to http://www.scylladb.com for upgrade instructions
```

### PR DESCRIPTION
**Pull request by syuu1228**
Introduce support nonroot and offline mode for scylla-housekeeping.

Closes #13084

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -489,6 +498,39 @@ StandardError=
 StandardError=inherit
 EOS
     fi
+    for i in daily restart; do
+        install -d -m755 "$rsystemd"/scylla-housekeeping-$i.service.d
```
Comment (avikivity) [inline install.sh:502]: Are these files present in rpm/deb installs? If so they need to be marked so the package manager can clean them up. Or better fold them into the main file.

Comment (syuu1228) [reply install.sh:502]: No, it will create these drop-in files only when install.sh manually executed (when --packaging is not specified).

**Comment by syuu1228** [comment]
@scylladb/scylla-maint ping

**Comment by denesb** [comment]
@syuu1228 please rebase because this PR is ancient and also to get a new run of CI (the last one is over a year old).

**Comment by syuu1228** [comment]
Rebased.

**Comment by fruch** [comment]
@scylladb/scylla-maint 

1) FYI, CI today doesn't cover setup/install of scylla, so merging it base on CI results doesn't really help.
2) there one comment from Avi more than a year old, no other review/approval from anyone

I would recommend asking for links of working tests (there are such test, even they are not part scylla-ci) in any case of changes to setup/install

**Pull request closed without merging**